pkg/controller/init.go

```diff
@@ -185,6 +185,10 @@ func (c *Controller) initNodeSwitch() error {
 	}
 
 	_, err = c.config.KubeOvnClient.KubeovnV1().Subnets().Create(context.Background(), &nodeSubnet, metav1.CreateOptions{})
+	if err != nil {
+		klog.Errorf("failed to create subnet %s: %v", c.config.NodeSwitch, err)
+		return err
+	}
```

Comment:
No need to return here. Or `return nil` in the following line.

Reply:
@zhangzujian You're right. I have changed them and the latest code has been commited.